lighthouse-core/test/gather/gather-runner.js

```diff
@@ -48,25 +48,16 @@ describe('GatherRunner', function() {
       }
     };
 
-    return GatherRunner.loadPage(driver, {}, {
```

Comment:
yes, third argument was supposed to be the second, but didn't fail because the old parameter checking code was adding `loadPage` to the empty object